src/bb-library/info.htm

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

```

Comment:
what is the purpose of this file?

Reply:
@fordnox It is a file from the maintainer that is letting people know more detailed information about it such as it requires ext-mbstring, what license it is using, along with contact information. I do not know why it was left or committed in the maintainers language. Do you want me to reinterpret in english or just remove the file.